daprdocs/content/en/operations/hosting/self-hosted/self-hosted-no-docker.md

```diff
@@ -51,16 +64,48 @@ INFO[0001] leader is established.                        instance=Nicoletaz-L10.
 
 ```
 
-From here on you can follow the sample example created for the [java-sdk](https://github.com/dapr/java-sdk/tree/master/examples/src/main/java/io/dapr/examples/actors), [python-sdk](https://github.com/dapr/python-sdk/tree/master/examples/demo_actor) or [dotnet-sdk]({{< ref "dotnet-actors-howto.md" >}}) for running an application with Actors enabled.
+{{% /codetab %}}
+
+{{% codetab %}}
+
+When running standalone Windows placement, specify port 6050:
```

Comment:
Should this say this?
```suggestion
When running standalone placement on Windows, specify port 6050:
```

Also would it be clearer if placement was marked as a code like this `placement` or not?